src/main/java/com/fasterxml/jackson/core/base/ParserMinimalBase.java

```diff
@@ -768,7 +769,13 @@ protected void reportUnexpectedNumberChar(int ch, String comment) throws JsonPar
     
     protected void _throwInvalidSpace(int i) throws JsonParseException {
         char c = (char) i;
-        String msg = "Illegal character ("+_getCharDesc(c)+"): only regular white space (\\r, \\n, \\t) is allowed between tokens";
+        String msg = "Illegal character ("+_getCharDesc(c)+"): only regular white space (\\r, \\n, \\t, \\u001E) is allowed between tokens";
```

Comment:
Given the note we have below, not sure if still need to include it here. I believe it's better to be removed given it's managed by a feature (not regular?)